### ISSUE TITLE
Hit boxes on allow scripts are too small, create unintended modal closure when missed.

### ISSUE THREAD

**Issue by alexwykoff** [brave/browser-laptop#7430]
- Did you search for similar issues before submitting this one?
Yes

- Describe the issue you encountered:
When attempting to click on the checkboxes to allow scripts, I missed by what felt like a pixel. The  modal closed.

- Platform (Win7, 8, 10? macOS? Linux distro?):
Tested on OS X

- Brave Version (revision SHA):
0.13.5 (0dc505a5d4ab6e67550e1a39f448a610e0497e39)

- Steps to reproduce:
    1. Block scripts for a site (or globally)
    2. Visit a site with scripts
    3. Open the script control modal on the urlbar
    4. Attempt to click on checkboxes quickly 
    
- Actual result:
If you miss the checkboxes by a minor amount, the modal closes and the user has to start over.

- Expected result:
The checkboxes should have some forgiveness to them and be contained in a margin of space which does not register clicks.

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
Yes

- Is this an issue in the currently released version?
No, this is a new feature.

- Can this issue be consistently reproduced?
Yes

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:

- Any related issues:
https://github.com/brave/browser-laptop/pull/7272

**Comment by NejcZdovc** [comment]
@alexwykoff I think that we shouldn't close dialog if you click inside the dialog. We should only close it if:
- you click on either button
- you click outside of the box

cc @bradleyrichter

**Comment by diracdeltas** [comment]
i'm working on that @NejcZdovc
